example_app_generator/generate_stuff.rb

```diff
@@ -44,6 +44,7 @@ def setup_tasks
 
     def final_tasks
       copy_file 'spec/verify_no_active_record_spec.rb'
+      copy_file 'spec/verify_no_fixture_setup_spec.rb'
```

Comment:
?

Reply:
Added another spec to the smoke app. Should it be done differently?

Reply:
Sorry no, I read this as being identical several times, must be getting later :joy:

Reply:
I was confused as well when I've finished typing its name 😆